metals/src/main/scala/scala/meta/internal/worksheets/WorksheetClassLoader.scala

```diff
@@ -8,7 +8,7 @@ package scala.meta.internal.worksheets
  * allowing the Metals server to call mdoc instances from different Scala
  * versions.
  */
-class MdocClassLoader(parent: ClassLoader) extends ClassLoader(null) {
+class MdocClassLoader(parent: ClassLoader) extends ClassLoader(parent) {
```

Comment:
So I think this one was on purpose in order not to bring in the Scala 2.12 classes from Metals server. Maybe let's use exactly what was used in Dotty?
```suggestion
class MdocClassLoader(parent: ClassLoader) extends ClassLoader(ClassLoader.getSystemClassLoader.getParent) {
```

Reply:
Ahh yes, I was just looking at the failures now. Let me try that.